Comments/Services/ConsultationListService.cs

```diff
@@ -73,14 +73,14 @@ public async Task<(ConsultationListViewModel consultationListViewModel, Validate
             if (model.InitialPageView) //set defaults here for the filters - this code might be called by the SSR or by the client-side.
 			{
 				//if unset (by querystring) set the default for the contribution filter. - the filter should be enabled (HasContributed), for regular users (i.e. not admins or team users) and the user has contributed to any.
-				if (model.Contribution == null)
-				{
-					var userHasCommentedOrAnswered = sourceURIsCommentedOrAnswered.Any();
-					if ((!isAdminUser && !isTeamUser) && userHasCommentedOrAnswered)
-					{
-						model.Contribution = new List<ContributionStatus> {ContributionStatus.HasContributed};
-					}
-				}
+				//if (model.Contribution == null)

```

Comment:
Can we remove this code instead of commenting it out please? Just keeps the code base cleaner unless there's a very good reason, then we should add a note as to why we're keeping it.